kola/tests/ignition/oem.go

```diff
@@ -91,7 +91,8 @@ func init() {
 
 // reusePartition asserts that even if the config uses a different fs format, we keep using `btrfs`.
 func reusePartition(c cluster.TestCluster) {
-	out := c.MustSSH(c.Machines()[0], `lsblk --output FSTYPE,LABEL,MOUNTPOINT --json | jq -r '.blockdevices | .[] | select(.label=="OEM") | .fstype'`)
+	// No idea why this sometimes returns two entries
+	out := c.MustSSH(c.Machines()[0], `lsblk --output FSTYPE,LABEL,MOUNTPOINT --json | jq -r '.blockdevices | .[] | select(.label=="OEM") | .fstype' | tail -n 1`)
```

Comment:
is there any way to output a warning if this happens and get the output of `lsblk --output FSTYPE,LABEL,MOUNTPOINT --json` so that a human can look over it?

Reply:
we could conditionally use `capnslog` like there: 
https://github.com/flatcar-linux/mantle/blob/13cba583a571c445975306723ffd5c70189267cd/kola/tests/bpf/bpf.go#L54

but if the test is always a success, we will never look at the output :/

Reply:
Yes, if we want to know what's going on we can add more printing here instead of silencing it

Reply:
Added it now